modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/edit_object_state.jsp

```diff
@@ -0,0 +1,32 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+ObjectDefinitionsStateManagerDisplayContext objectDefinitionsStateManagerDisplayContext = (ObjectDefinitionsStateManagerDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+ObjectField objectField = (ObjectField)request.getAttribute(ObjectWebKeys.OBJECT_FIELD);
+%>
+
+<react:component
+	module="js/components/EditObjectStateField"
+	props='<%=
+		HashMapBuilder.<String, Object>put(
+			"objectField", objectDefinitionsStateManagerDisplayContext.getObjectFieldJSONObject(objectField)
+		).build()
```

Comment:
Hi @cgmonte,

Could you define the `objectField` variable inline, e.g.:

 ````
 objectDefinitionsStateManagerDisplayContext.getObjectFieldJSONObject(
 (ObjectField)request.getAttribute(ObjectWebKeys.OBJECT_FIELD))

 ````
 
 Thank you!